daemons/pacemakerd/pacemakerd.h

```diff
@@ -20,6 +20,7 @@ typedef struct pcmk_child_s {
     const char *uid;
     const char *command;
     const char *endpoint;  /* IPC server name */
+    bool needs_corosync;
```

Comment:
Should this be renamed to something more cluster-generic?  It's not public so we could always rename it later, but still.

Reply:
Yes, needs_cluster would be better

Reply:
What we're going for is "needs the cluster messaging layer to talk to peers on other nodes". For most of the daemons that's the only thing from the cluster layer that they use. The controller also uses quorum information from the cluster layer, and pacemakerd uses the configuration interface, but I think we can gloss over that for these purposes.